RoaringBitmap/src/main/java/org/roaringbitmap/longlong/Roaring64NavigableMap.java

```diff
@@ -692,6 +692,53 @@ private int highestHigh() {
     return RoaringIntPacking.highestHigh(signedLongs);
   }
 
+  /**
+   * to be used with naivelazyor
+   */
+  public void repairAfterLazy() {
+    for (Entry<Integer, BitmapDataProvider> e2 : highToBitmap.entrySet()) {
+      BitmapDataProvider lowBitmap2 = e2.getValue();
+      if (lowBitmap2 instanceof RoaringBitmap) {
+        ((RoaringBitmap) lowBitmap2).repairAfterLazy();
+      } else {
+        throw new UnsupportedOperationException(
+                ".repairAfterLazy is not supported for " + lowBitmap2.getClass());
+      }
+    }
+  }
+  /**
+   * In-place bitwise OR (union) operation without maintaining cardinality.
+   * Don't forget to call repairAfterLazy() afterward. The current bitmap is modified.
+   *
+   * @param x2 other bitmap
+   */
+  public void naivelazyor(final Roaring64NavigableMap x2) {
+    boolean firstBucket = true;
+    for (Entry<Integer, BitmapDataProvider> e2 : x2.highToBitmap.entrySet()) {
+      // Keep object to prevent auto-boxing
+      Integer high = e2.getKey();
+
+      BitmapDataProvider lowBitmap1 = this.highToBitmap.get(high);
```

Comment:
It may be over-looked optimisation, but we may prefer a double-iteration through entrySets, not to have these indirections

Reply:
but I see other method(or xor....) implement like this style.

Reply:
I suppose I was lazy at given point of time :trollface: . Was fine anyway, even finer if consistent with the rest of the class

Reply:
I think this style is more simple and readable, so make it  consistent with others.